nixos/doc/manual/installation/installing.xml

```diff
@@ -108,8 +108,9 @@
 > 1      # <lineannotation>(make it a partition number 1)</lineannotation>
 >        # <lineannotation>(press enter to accept the default)</lineannotation>
 >        # <lineannotation>(press enter to accept the default and use the rest of the remaining space)</lineannotation>
-> a      # <lineannotation>(make the partition bootable)</lineannotation>
 > x      # <lineannotation>(enter expert mode)</lineannotation>
+> A      # <lineannotation>(make the partition bootable)</lineannotation>
+> 1      # <lineannotation>(...partition one, the boot partition)</lineannotation>
```

Comment:
Was partition 1 no the root partition in this example?

Reply:
good catch, this is probably my typo. I was copying from line 108 above which was the last partition mentioned. Probably should be partition 3.